lib/IRGen/TypeInfo.h

```diff
@@ -148,7 +148,6 @@ class TypeInfo {
            IsABIAccessible_t abiAccessible,
            SpecialTypeInfoKind stik) : StorageType(Type) {
     assert(stik >= SpecialTypeInfoKind::Fixed || !alwaysFixedSize);
-    assert(!A.isZero() && "Invalid alignment");
```

Comment:
Why isn't it useful to keep this anymore?

Reply:
Because it was defined away and removed from the type. All alignments are >= 1 now. If somebody wants to express non-alignment, they can use `llvm::Optional. This was covered at the top of the pull request and in the commit message.